src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -849,8 +849,13 @@ public WriteMethod writeMethod() {
   private static class UrlListValidator implements Validator {
 
     @Override
+    @SuppressWarnings("unchecked")
     public void ensureValid(String name, Object value) {
-      @SuppressWarnings("unchecked")
+      if (value == null) {
+        throw new ConfigException(
+            name, value, "The provided url list is null."
+        );
```

Comment:
might fit on one line
```suggestion
        throw new ConfigException(name, value, "Url list must be provided and non-null.");
```